runtime/queries/smali/injections.scm

```diff
@@ -0,0 +1 @@
+(comment) @comment
```

Comment:
```suggestion
((comment) @injection.content
 (#set! injection.language "comment"))
```

we follow tree-sitter for injections https://tree-sitter.github.io/tree-sitter/syntax-highlighting#language-injection, nvim has its own injection format

Reply:
I made the change as suggested, and this reminded me to check `goto_next_comment` which I found doesn't work.

tree-sitter-subtree does report `(comment)` for comment lines, so I don't know what to check beyond that